proposals/lip-0054.md

```diff
@@ -67,9 +67,13 @@ This command is used to terminate a sidechain that violated the liveness conditi
 
 ### Message Recovery from the Sidechain Channel Outbox
 
-This mechanism allows to recover any CCM pending in the sidechain channel outbox. That is, those CCMs sent from the sidechain that have not been included in the (terminated) receiving chain yet. Specifically, this includes all the CCMs whose indices (the position of these CCMs in the outbox tree) are larger than the index of the last message that the receiving sidechain reported to have included in its inbox. This recovery mechanism requires these conditions to work as follows:
+This mechanism allows to recover any CCM pending in the sidechain channel outbox. That is, those CCMs sent from the sending chain that have not been included in the (terminated) receiving chain yet. Specifically, this includes all the CCMs whose indices (the position of these CCMs in the outbox tree) are larger than the index of the last message that the receiving sidechain reported to have included in its inbox. 
 
-* The sidechain chain account has been terminated on the mainchain, i.e. there exists an entry in the terminated chain substore with store key equal to the sidechain chain ID. This entry is created automatically on the mainchain.
+Notice that the message recovery mechanism requires that the channel outbox is stored in the chain where the commands are processed. In the SDK 6, sidechain channels can only be stored on the mainchain. This means that the message recovery mechanism would only work on the mainchain.
+
+The message recovery mechanism requires the following conditions:
+
+* The sidechain chain account has been terminated on the mainchain, i.e. there exists an entry in the terminated state substore with store key equal to the sidechain chain ID. This entry is created automatically on the mainchain.
```

Comment:
the entry could be in the terminated state substore of a sidechain in the case of direct channels tho.

Reply:
I am not sure I understand (it does not matter for this LIP since you know removed this point just to clarify), but do you mean that in case of direct channels you would require the sidechain account to be terminated in mainchain? I don't think this is consistent.  

It's good that this point is removed because it was confusing ;-)